lua/nvim-treesitter/parsers.lua

```diff
@@ -47,6 +47,7 @@ for ft, lang in pairs {
   mysql = "sql",
   sbt = "scala",
   neomuttrc = "muttrc",
+  api = "goctl",
```

Comment:
```suggestion
```
This is for additional supported filetypes.

Reply:
> This is for additional supported filetypes.

But the official use is.api file as the end, how should I design.

Reply:
These are filetypes, not extensions. First you'll have to get "goctl" filetype detection into Vim, then (if the filetype is different from the language name) add a mapping here. But this plugin does not do filetype detection.